src/prefect/server/services/flow_run_notifications.py

```diff
@@ -151,6 +152,7 @@ def get_ui_url_for_flow_run_id(self, flow_run_id: UUID) -> str:
         Args:
             flow_run_id: the flow run id.
         """
+        return urls.url_for(flow_run_id=flow_run_id)
         ui_url = PREFECT_UI_URL.value() or "http://ephemeral-prefect/api"
         return f"{ui_url}/flow-runs/flow-run/{flow_run_id}"
```

Comment:
```suggestion
```

Reply:
I had to do a little more to fix this line, which I'd forgotten about, so thanks for reminding me!